src/www/app/server/repository.ts

```diff
@@ -203,21 +116,21 @@ export class OutlineServerRepository implements ServerRepository {
     }
```

Comment:
this "validateAccessKey" logic I think I'm going to move into the new class. the only part that should stay in the OutlineServerRepository is the `alreadyAddedServer` clause, which should be its own method